.editorconfig

```diff
@@ -10,5 +10,5 @@ charset = utf-8
 trim_trailing_whitespace = true
 insert_final_newline = true
 
-[package.json]
+[{package*.json,*.yml}]
```

Comment:
Not sure if you prefer all JSON files to stick with 2 spaces, but I left it to just the package files for now